keyboards/heliotrope/info.json

```
@@ -0,0 +1,104 @@
+{
+    "manufacturer": "MATTMCCA",
+    "keyboard_name": "heliotrope",
+    "maintainer": "MATTMCCA",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+	    "rgblight": true
+    },
+    "rgblight": {
+        "led_count": 61,
+        "pin": "A4",
+        "hue_steps": 10,
+		"max_brightness": 255,
+        	"animations": {
+			"all": true
```

Could you explicitly define which animations are enabled? 

```suggestion
            "alternating": true,
            "breathing": true,
            "christmas": true,
            "knight": true,
            "rainbow_mood": true,
            "rainbow_swirl": true,
            "rgb_test": true,
            "snake": true,
            "static_gradient": true,
            "twinkle": true
```

---

changed as requested.